src/styles/auto-prefix.js

```diff
@@ -20,18 +20,29 @@ export default {
     let prefixer = prefixers[userAgent];
     // None found, create a new instance
     if (!prefixer) {
-      prefixer = new InlineStylePrefixer(userAgent);
+      prefixer = new InlineStylePrefixer({userAgent: userAgent});
```

Comment:
@newoga Thanks for raising this eslint discussion :+1:.
I completely agree with https://github.com/airbnb/javascript#3.5.
But for https://github.com/airbnb/javascript#3.6, I'm not sure, I kind of prefer the old way.
What do you think?